tests/unit_tests/test_cli.py

```diff
@@ -626,3 +626,228 @@ def test_cli_debug_log_scoping(runner, test_settings):
                 importlib.reload(cli)
                 assert cli._username == test_user
                 assert cli._wandb_log_path.endswith(f"debug-cli.{test_user}.log")
+
+
+class TestParseServiceConfig:
```

Comment:
We don't usually group tests this way in the SDK: tests functions should be at the top level. +1 to creating a separate test file for these.